Rakefile

```diff
@@ -14,12 +16,25 @@ end
 
 namespace :design_system do
   desc "All Design System Tests"
-  task :all do
+  task all: [:report_folders] do
```

Comment:
as this isn't enumerable, lets leave it as `all: :reports_folders do` that way it's slightly more optimal.

Reply:
Done